plutus-core/untyped-plutus-core/src/UntypedPlutusCore/Transform/Inline.hs

```diff
@@ -498,10 +507,10 @@ inlineSaturatedApp t
           fullyApplyAndBetaReduce varInfo args >>= \case
             Nothing -> pure t
             Just fullyApplied -> do
+              rhs <- liftDupable (let Done rhs = _varRhs varInfo in rhs)
```

Comment:
The renaming of the RHS wasn't done before, because it wasn't an `InlineTerm`, but I think it should be done.